src/index.js

```diff
@@ -59,11 +59,11 @@ module.exports = ({ types: t }) => {
       );
 
       const explicitRequires = dependencyParameterPairs
-        .filter(([dependency]) => {
-          return !t.isStringLiteral(dependency) || !keywords.includes(dependency.value);
-        })
         .map(([dependency, paramName]) => {
           return createRequireExpression(dependency, paramName);
+        })
+        .filter(requireExpression => {
+          return requireExpression; // Remove 'undefined' expressions
```

Comment:
Let's change the condition to `requireExpression !== undefined` and I think we can remove the comment.